test_runner/batch_others/test_gc.py

```diff
@@ -47,8 +47,8 @@ def test_gc(zenith_cli, pageserver, postgres, pg_bin):
                     print("GC duration {elapsed} ms, relations: {n_relations}, dropped {dropped}, truncated: {truncated}, deleted: {deleted}".format_map(row))
                     assert row['n_relations'] == n_relations
                     assert row['dropped'] == 0
-                    assert row['truncated'] == 1
-                    assert row['deleted'] == 1
+                    assert row['truncated'] == 30
```

Comment:
Why did this change?

Reply:
It is different version of garbage collector (it uses single iterator though all objects, rather that nested loop for-all-relations/for-all-blocks) which slightly differently calculates this metrics. It is the source of this difference.